spanner/errors.go

```diff
@@ -26,6 +27,11 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var (
```

Comment:
nit: Could this could be `const`?

Reply:
No, const is reserved for string types only, here it is of error type.

Reply:
ack